src/block.js

```diff
@@ -106,172 +61,35 @@ export default registerBlockType(
-					</InspectorControls>
+					<Inspector 
+						{ ...{ onChangeTweet, onChangeButton, onChangeAnimation, onChangeDuration, onChangeDelay, toggleInfinite,...props} }
+					/>
 				),
+
+				// Toolbar
 				!! props.focus && (
-					<BlockControls key="toolbar">
-						<Toolbar>
-							<IconButton
-								icon={ bbutton }
-								label={ __( 'Big Button' ) }
-								onClick={ () => onChangeTheme( 'bbutton' ) }
-								className={
-									classnames(
-										{ 'tld-selected-icon': ( props.attributes.theme === 'bbutton' ) },
-									)
-								}
-							/>
-							<IconButton
-								icon={ dashed }
-								label={ __( 'Dashed' ) }
-								onClick={ () => onChangeTheme( 'dashed' ) }
-								className={
-									classnames(
-										{ 'tld-selected-icon': ( props.attributes.theme === 'dashed' ) },
-									)
-								}
-							/>
-							<IconButton
-								icon={ minimalist }
-								label={ __( 'Minimalist' ) }
-								onClick={ () => onChangeTheme( 'minimalist' ) }
-								className={
-									classnames(
-										{ 'tld-selected-icon': ( props.attributes.theme === 'minimalist' ) },
-									)
-								}
-							/>
-						</Toolbar>
-						<Toolbar>
-							<DropdownMenu
-								icon="editor-textcolor"
-								label={ __( 'Font' ) }
-								menuLabel={ __( 'Font' ) }
-								controls={ [
-									{
-										title: __( 'Poiret One' ),
-										icon: 'editor-textcolor',
-										onClick: () => onChangeFont( 'poiret-one' ),
-									},
-									{
-										title: __( 'Lobster Two' ),
-										icon: 'editor-textcolor',
-										onClick: () => onChangeFont( 'lobster-two' ),
-									},
-									{
-										title: __( 'Raleway' ),
-										icon: 'editor-textcolor',
-										onClick: () => onChangeFont( 'raleway' ),
-									},
-									{
-										title: __( 'Titillium Web' ),
-										icon: 'editor-textcolor',
-										onClick: () => onChangeFont( 'titillium-web' ),
-									},
-									{
-										title: __( 'Indie Flower' ),
-										icon: 'editor-textcolor',
-										onClick: () => onChangeFont( 'indie-flower' ),
-									},
-								] }
-							/>
-						</Toolbar>
-					</BlockControls>
+					<Controls {... { onChangeTheme, onChangeFont,...props }} />
 				),
+
+				// Edit UI
 				<div className={ props.className } key={ props.className }>
 					<div
 						id="tld-actt-tweet-container"
 						className={ classnames(
-							boxTheme,
-							fontStyle,
-							{ animated: !! props.focus && ( props.attributes.animaion !== 'none' ) },
+							`tld-actt-${props.attributes.theme}`,
+							props.attributes.font,
+							{ animated: !! props.focus && ( props.attributes.animation !== 'none' ) },
 							animation,
 							{ infinite: !! props.focus && ( props.attributes.infinite ) },
 						) }
-						style={ animationStyle }
+						style={	{ 'animation-duration': `${props.attributes.duration}s`, 'animation-delay': `${props.attributes.delay}s` } }
 					>
 						<Editable
```

Comment:
@ajitbohra While you're at it, they have replaced `Editable` with `RichText` so you can replace that as well: https://github.com/WordPress/gutenberg/blob/f889d2791510a61ca410da91f7c93b739304c10b/blocks/rich-text/editable.js#L6

Not sure if that's out in production at WPorg so make sure to test. :D

Reply:
Yes, Editable will be deprecated in the upcoming release in favor of RichText. Not sure if it's available in production.

Reply:
@HardeepAsrani RichText is not yet available in production added "ToDo" comment